### ISSUE TITLE
Bad performance on customized robotwin task

### ISSUE THREAD

**Issue by KevinGeng3** [Robbyant/lingbot-va#65]
Hi, your work is fantastic! But when I collected about 100 episodes on my customized robotwin task and finetuned the model, I found that the model's performance does not seem to be very good.

I almost didn't change the default training parameters, except for va_robotwin_cfg.norm_stat, which I changed to the actual values ​​from my data.

# Training parameters
va_robotwin_train_cfg.load_worker = 16
va_robotwin_train_cfg.save_interval = 2000
va_robotwin_train_cfg.gc_interval = 50
va_robotwin_train_cfg.cfg_prob = 0.1

va_robotwin_train_cfg.learning_rate = 1e-5
va_robotwin_train_cfg.beta1 = 0.9
va_robotwin_train_cfg.beta2 = 0.95
va_robotwin_train_cfg.weight_decay = 0.1
va_robotwin_train_cfg.warmup_steps = 10
va_robotwin_train_cfg.batch_size = 1 
va_robotwin_train_cfg.gradient_accumulation_steps = 1
va_robotwin_train_cfg.num_steps = 10000 

I trained the model on 4*A800 and here is the loss curve:

<img width="3096" height="1558" alt="Image" src="https://github.com/user-attachments/assets/65b6cef5-309b-48a6-8da4-dabfe4aefd88" />

The model performs as follows:

https://github.com/user-attachments/assets/bedc4cb2-6216-4598-9c6c-a24ea5b32f04

I would be very grateful if anyone could help me. Thank you!

**Comment by zqh0253** [comment]
Thanks for your interest. I'd suggest checking the following: 
1. First, run a TI2V inference to verify the generated video looks normal. 
2. If (1) looks fine, the issue is likely with the IDM. In particular, I notice that the first several frames are entirely blank — no robot arms are visible. This is ill-posed, as it requires the IDM to learn a
  one-to-many mapping (the same black image mapped to different robot motions).

**Comment by KevinGeng3** [comment]
> Thanks for your interest. I'd suggest checking the following:感谢您的关注。我建议您检查以下几点：
> 
> 1. First, run a TI2V inference to verify the generated video looks normal.首先，运行 TI2V 推理以验证生成的视频是否正常。
> 2. If (1) looks fine, the issue is likely with the IDM. In particular, I notice that the first several frames are entirely blank — no robot arms are visible. This is ill-posed, as it requires the IDM to learn a如果(1)看起来没有问题，问题可能出在 IDM 上。特别是，我注意到前几个帧完全是空白的——看不到任何机械臂。这是不合理的，因为它要求 IDM 学习一个
>    one-to-many mapping (the same black image mapped to different robot motions).一对多映射（相同的黑色图像映射到不同的机器人动作）。

Thank you for your reply! 
First of all, I checked the TI2V inference and made sure there were no problems. 
Then, to address the issue of the robotic arm not being visible, I readjusted the robotic arm's configuration. The following is the current result:

https://github.com/user-attachments/assets/94a7035c-2b8a-401f-a4d0-daa71d1c6c89

<img width="3506" height="1558" alt="Image" src="https://github.com/user-attachments/assets/cd3f993c-03b9-48e6-82ea-a203ebd0db31" />

I currently suspect it's a training issue. For 100 episodes, I used 4*A800 for training, and both va_robotwin_train_cfg.batch_size and va_robotwin_train_cfg.gradient_accumulation_steps are set to 1. And va_robotwin_train_cfg.num_steps is 6k. I suspect that even though the loss has been reduced low enough, the number of training steps is still too small. 

I'm not sure if you have any experience here, but I hope you can answer my question. Thank you.

**Comment by KevinGeng3** [comment]
I discovered the problem was that I forgot to change the attn_mode in transformer/config.json from flash_attn to flex during training. This configuration has a significant impact on training, so please remember to confirm.

**Comment by XiaoJichun** [comment]
Could you please tell me if you have ever tried to combine multiple tasks into one model and verify the results? Thank you very much for sharing.
